Sources/KituraContracts/Contracts.swift

```diff
@@ -561,13 +565,18 @@ public protocol QueryParams: Codable {
     static var dateEncodingStrategy: JSONEncoder.DateEncodingStrategy { get }
 }
 
+/// Defines default values for the `dateDecodingStrategy` and `dateEncodingStrategy`. The
+/// default formatting for a `Date` in a `QueryParams` type is defined by `Coder.dateFormatter`,
+/// which uses the "UTC" timezone and "yyyy-MM-dd'T'HH:mm:ssZ" date format.
 extension QueryParams {
 
-    static var dateDecodingStrategy: JSONDecoder.DateDecodingStrategy {
+    /// Default value: `Coder().dateFormatter`
+    public static var dateDecodingStrategy: JSONDecoder.DateDecodingStrategy {
         return .formatted(Coder().dateFormatter)
```

Comment:
This creates a new `Coder` every time the property is accessed - is that intentional?